mail_tracking/models/mail_tracking_email.py

```diff
@@ -43,7 +43,7 @@ class MailTrackingEmail(models.Model):
         string="Date", readonly=True, compute="_compute_date", store=True
     )
     mail_message_id = fields.Many2one(
-        string="Message", comodel_name="mail.message", readonly=True, index=True
+        string="Mail Message", comodel_name="mail.message", readonly=True, index=True
```

Comment:
just to let you know in such case it would be the default string generated by odoo so we could do

```suggestion
        comodel_name="mail.message", readonly=True, index=True
```

cf: https://github.com/odoo/odoo/blob/14.0/odoo/fields.py#L398-L403
```python
        if not self.string and not self.related:
            # related fields get their string from their parent field
            self.string = (
                name[:-4] if name.endswith('_ids') else
                name[:-3] if name.endswith('_id') else name
            ).replace('_', ' ').title()
```

Reply:
Nice! I did not catch that. Thank you for the heads up 🙏 
I'm applying this suggestion as I don't think an explicit label is required here.